### PR TITLE
readme: expanded overview and installation sections

### PR DESCRIPTION
**Pull request by jtsiomb**
Proposed expanded readme text, explaining in some more detail what freeglut is all about, what it does, and what are our goals. It also expands the installation section slightly, explaining that we're using cmake, and also pointing towards the unsupported alternative build files directory.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,9 +48,16 @@ and build instructions.
 INSTALLATION
 ------------
 
-See README.cmake as a starting point, as well as the other README
+Freeglut relies on cmake (minimum 3.0.0) to generate build files for all the
+different supported platforms and toolchains.
+
+See `README.cmake` as a starting point, as well as the other `README`
```
Comment (dcnieho) [inline README.md:54]: turn into link using `[link text](README.cmake)`

File: README.md [hunk 2]
```diff
@@ -31,9 +48,16 @@ and build instructions.
...
 files for further info.
 
+For those cases where using cmake is inconvenient, we also provide a set of
+unofficial, unsupported, possibly unmaintained, but potentially useful,
+alternative build files. See `altbuild/README.md` for details.
```
Comment (dcnieho) [inline README.md:59]: `[link text](altbuild/README.md)`

**Comment by dcnieho** [comment]
I just realized i can make edits myself, made a small further adjustment to make it look nicer in my opinion :)

**Comment by jtsiomb** [comment]
merged manually, thanks for the comments and edits.

**Pull request closed without merging**